### ISSUE TITLE
Suggest type signature throwing exception

### ISSUE THREAD

**Issue by OmanF** [carymrobbins/intellij-haskforce#204]
(Maybe related to #113 ?)
When trying to use the suggest type (alt+equals) I get the following error:
```
ghc-modi errorghc-modi
execution was aborted:
java.util.concurrent.ExecutionException:
java.util.InputMismatchException
```

When hitting the configuration button and taken to the page to choose the tools, I use the auto-find function to find the tools, specifically ghc-modi. All the tools are valid (I get the details line auto-filled with the correct details. Also, all tools are in the ~/.cabal/bin folder so I know they are valid).

Some tech data taken from Help -> About:
IntelliJ IDEA 15.0 (build #IU-143.381)
JRE: 1.8.0_60-b27 amd64

Thanks you

**Comment by carymrobbins** [comment]
What version of ghc, ghc-mod, and ghc-modi are you running?  For what it's worth, I'm currently working on ghc-mod 5.4 support (https://github.com/carymrobbins/intellij-haskforce/issues/182) and should be able to get that completed within the next week.

**Comment by OmanF** [comment]
I'm running GHC 7.10.2 and ghc-mod/ghc-modi at 5.4.0.0
(BTW, I'm watching the thread about 182 keenly. Great news about the upcoming update).

Thanks!

**Comment by carymrobbins** [comment]
@OmanF - The latest release now supports ghc-mod 5.4, so this may have gone away for you now.  Try it again with the new release and let me know if you are still experiencing the issue.

**Comment by OmanF** [comment]
Sorry for not answering for a few days, I was computer-deprived.

To the point, the same error message is thrown!
I've updated the plugin to the latest version, auto-found GHC-MODi to be GHC-MOD with the interactive legacy flag.

I'm also now on IntelliJ 15.0.1.
Same JRE, GHC and ghc-mod/modi modules.

**Comment by carymrobbins** [comment]
I've been able to reproduce this and will have it resolved for the next release.

**Comment by carymrobbins** [comment]
Thanks for the report, this will be resolved in the next release, **v0.3-beta.25**.  Please reopen if the issue is not resolved for you.

**Comment by OmanF** [comment]
Original error resolved. Now, when trying to show type info **for the first time**, getting the following exception thrown:

ghc-modi error
         Killing ghc-modi due to process failure.
         You can restart it using Tools > Restart ghc-modi
         Executing ghc-modi command 'type /home/ohadf/Projects/Haskell/myTestHaskellProject/src/MyTest.hs 3 3' failed with error: EXCEPTION: types:
         No instance for (Enum a0) arising from a use of ‘succ’
         The type variable ‘a0’ is ambiguous
         Relevant bindings include
         mySucc :: a0 -> a0 (bound at src/MyTest.hs:3:1)
         Note: there are several potential instances:
         instance Enum Ordering -- Defined in ‘GHC.Enum’
         instance Enum Integer -- Defined in ‘GHC.Enum’
         instance Enum () -- Defined in ‘GHC.Enum’
         ...plus six others

**Also**, after clearing the exception (hitting the "x" on the exception pop-up windows), choosing the menu item *View -> Type info* does **not** work, but hitting the keyboard's *alt + equals* works but returns *No type found* for all attempts.
(All other menu entries, including such entries as reformat haskell code, works like charm).
